### ISSUE TITLE
Premières idées

### ISSUE THREAD

**Issue by bluepioupiou** [jobot0/stack-bingo#1]
* Choix de : technologie back, technologie front, format d’échange, type de container, environment de livraison,hébergement de code ? Outil de ci/cd ?
* On pourrait faire une sorte de tirage au sort à la bandit manchot. Choix de fixer une ou plusieurs techno ou de de sélectionner celles qui peuvent être tirée au sort ou de laisser complètement au hasard. 
* Petite fiche technique sur chaque techno : avantages, inconvénients, classification, courbe d'apprentissage etc (comme les joueurs de foot un graph étoile)
* chaque ensemble de techno pourra avoir des liens vers des repos d’exemple et des vote sur quelle solution est la plus simple t et élégante en bootstrap sur la communauté 
* a mettre en avant pour les écoles et pourquoi pas les entretiens

**Comment by jobot0** [comment]
Yes carrément je pense que les 2 premiers points seraient une v1.0.0 (histoire d'avoir un core solide). Et les autres pourraient venir s'ajouter au fur et à mesure ?
